Packages/com.chisel.editor/Chisel/Editor/Editor/Scene/Handles/PointDrawing.cs

```diff
@@ -214,7 +260,15 @@ public static void Do(int id, Rect dragArea, ref List<Vector3> points, out Matri
                         GUIUtility.hotControl != id)
                         break;
 
-                    UpdatePoints(points, GetPointAtPosition(evt.mousePosition, dragArea));
+
+                    if (s_MouseJumping)
+                        s_MousePosition += evt.delta;
+                    else
+                        s_MousePosition = Event.current.mousePosition;
```

Comment:
When SetWantsMouseJumping is set to 1 (true) then the mouse can loop around the screen. But this only works if we keep track of the relative mouse position versus the current absolute mouse position